spec.html

```diff
@@ -120,36 +138,94 @@ <h1>Array.prototype.spliced ( _start_, _deleteCount_, ..._items_ )</h1>
                 <p>When the *spliced* method is called, the following steps are taken:</p>
 
                 <emu-alg>
-                    1. Let _O_ be ? ToObject(*this* value).
-                    1. Let _A_ be ArrayClone(_O_).
-                    1. Perform ? Call(Array.prototype.splice, _A_, «_start_, _deleteCount_, ..._items_»).
+                    1. Let _O_ be the *this* value.
+                    1. Let _len_ be ? LengthOfArrayLike(_O_).
+                    1. Let _relativeStart_ be ? ToIntegerOrInfinity(_start_).
+                    1. If _relativeStart_ is -&infin;, let _actualStart_ be 0.
+                    1. Else if _relativeStart_ &lt; 0, let _actualStart_ be max(_len_ + _relativeStart_, 0).
+                    1. Else, let _actualStart_ be min(_relativeStart_, _len_).
+                    1. If _start_ is not present, then
+                        1. Let _insertCount_ be 0.
+                        1. Let _actualDeleteCount_ be 0.
+                    1. Else if _deleteCount_ is not present, then
+                        1. Let _insertCount_ be 0.

```

Comment:
this entire algorithm reminds me that splice is terrible, god i hate it

Reply:
It's the Swiss Army knife of methods if ever there was one. It also reminds me of my old film school days were had to painstakingly edit films by physically splicing them together.